tests/results/DiffTest.py

```diff
@@ -279,3 +279,17 @@ def test_delete(self):
         self.uut.delete = True
         self.assertEqual(self.uut.modified, [])
         self.uut.delete = False
+
+    def test_diff_creation(self):
+        uut = Diff(create='create_test.py')
+        self.assertEqual(uut.create, 'create_test.py')
+        self.assertEqual(uut._file, [])
+        uut.add_lines(0, ['line2', 'line3'])
+        self.assertEqual(uut.modified, ['line2', 'line3'])
```

Comment:
I don't think I am changing the unified diff :/ is a diff for a new file not like just additions?